File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
# 총 테캐 갯수 5
# 과목수 + 점수를 리스트 입력으로 받는다
# 평균 낸 뒤에 평균 이상만 cnt하여 학생수 나눈다 --> 그걸로 % 구함

import sys
input = sys.stdin.readline

T = int(input())

for _ in range(T):
    scores = list(map(int, input().split()))
    cnt = 0
    avg = (sum(scores[1:])) / scores[0]

    for s in scores[1:]:
        if s >= avg:
            cnt += 1

    answer =  (cnt / scores[0]) * 100
    print(f'{answer : .3f}%') 

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int C = Integer.parseInt(br.readLine());
		for (int i = 0; i < C; i++) {
			String[] items = br.readLine().split(" ");
			double res = my_calculator(items);
			System.out.printf("%.3f%%\n", res);
		}
	}
	
	
	
	
	public static double my_calculator(String[] arr) {
		double avg = 0;
		int sum = 0;
		int N = Integer.parseInt(arr[0]);
		for (int i = 1; i < N+1; i++) {
			sum += Integer.parseInt(arr[i]);
		}
		avg = (double) sum / N;
		
		
		double ratio = 0;
		int cnt = 0;
		for (int i = 1; i < N+1; i++) {
			int temp_val = Integer.parseInt(arr[i]);
			if (temp_val > avg) cnt++;
		}
		ratio = (double) cnt / N * 100;
		return ratio;
	}
	
	
	
	
	
}